.github/workflows/build.yml

```diff
@@ -9,11 +9,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up JDK 11
+      - name: setup jdk 11
```

Comment:
Would there be a possibility to add a job for java 17 or 20 so we can find any potential migration problems before we actually decide to support them? I think it would be easier to find and fix problems as we go during PRs rather than accumulating them to fix at a later time

Reply:
yes .. that would be absolutely great - i'm interested in converting to gitactions and deprecating the jenkins server .. but not sure how feasable that is ... there hasn't been a meaningful test run on gitactions yet .. :P